### ISSUE TITLE
Move corresponding[Use]Element members to main SVGElement definition?

### ISSUE THREAD

**Issue by dstorey** [w3c/svgwg#440]
correspondingElement and correspondingUseElement were previously on the SVGElementInstance interface. This was converted to a mixin in SVG2. 

However, this mixin is only included on SVGElement. The [WebIDL spec recommends](https://heycam.github.io/webidl/#using-mixins-and-partials) to use a partial interface  if it is not mixed in to multiple interfaces. 

However, as the mixin is defined in the same spec as SVGElement, if we do want to move them then we should just put them directly in the main SVGElement definition.

@AmeliaBR thoughts? Is there a chance we'd want to mix this into another interface so it is still beneficial to be a mixin?

**Comment by AmeliaBR** [comment]
Defining it separately as a mixin was mostly about persisting the `ElementInstance` name from SVG 1.1, so there was some clear continuity. A partial interface would be fine. Or just adding to the main interface, although these features should definitely be considered At Risk right now, so I think there is benefit in keeping the spec "quarantined" in the same chapter as the rest of the `<use>` element definitions.

**Comment by dstorey** [comment]
Yeah, as no browsers support them (except Edge but on the old exposed interface rather than a mixin), we can reorganize them if there is any commitment to implement them.

**Comment by boggydigital** [comment]
Not blocking updated 2.0 CR publication - assigning 2.0 Recommendation milestone to clean this up before 2.0 REC
